src/tools/genVersionHeader.pl

```diff
@@ -29,7 +29,11 @@
 our $opt_N = 'VCSVERSION';
 our $opt_V = $now;
 
+our $opt_T = 'VCSREVISION';
+our $opt_C = $now;
```

Comment:
Can `$opt_T` and `$opt_C` be removed or renamed?  `$opt_` suggests that these are CLI arguments, but the call to `getopts()` isn't being modified.